src/transport/CASESession.cpp

```diff
@@ -31,16 +31,45 @@
 #include <core/CHIPEncoding.h>
 #include <core/CHIPSafeCasts.h>
 #include <protocols/Protocols.h>
+#include <support/BufferWriter.h>
 #include <support/CHIPMem.h>
 #include <support/CodeUtils.h>
 #include <support/SafeInt.h>
 #include <transport/SecureSessionMgr.h>
 
 namespace chip {
 
+// Remove Later
+static P256ECDHDerivedSecret fabricSecret;
+
+constexpr uint8_t kIPKInfo[] = { 0x49, 0x64, 0x65, 0x6e, 0x74, 0x69, 0x74, 0x79, 0x50, 0x72, 0x6f,
```

Comment:
Are these hardcoded values temporary? Or, is it something that's defined in CHIP specifications?